pkg/task/common.go

```diff
@@ -175,6 +184,9 @@ func DefineCommonFlags(flags *pflag.FlagSet) {
 	flags.Bool(flagEnableOpenTracing, false,
 		"Set whether to enable opentracing during the backup/restore process")
 
+	flags.BoolP(flagNoCreds, "", false, "Don't load credentials")
+	flags.BoolP(flagSkipCheckPath, "", false, "Skip path verification")
```

Comment:
for now please mark these two flags "Hidden" (see line 170 above) since afaik they are only used in tests.

Reply:
okay, they are marked as hidden now